site/app/models/gradeable/Gradeable.php

```diff
@@ -2005,4 +2005,10 @@ public function isUsingUploadedConfig(): bool {
 
         return !(strpos($this->getAutogradingConfigPath(), $config_upload_path) === false);
     }
+    public function setFilesToOmit() {
```

Comment:
Are theses empty for a reason?

Reply:
How we handle gradeable properties means I need setter and getter functions, otherwise an error is thrown. They are never actually used

Reply:
Jumping in a little late, but is there a reason that the value for `filesToOmit` isn't stored as a class variable here?

Reply:
In the database, I added a table that contains each wildcard, the gradable, and the level of permission that cannot see the file (only 4 rn). I think this a more versatile implementation that allows for easy changing down the line. However, it does mean that we have to handle filesToOmit differently, so storing it as a class variable wouldn't add anything (I think)